From 020992bb4cf2a3c58396906b6c0f5cb2bca9dca4 Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Thu, 9 Jun 2016 15:11:33 -0400
Subject: [PATCH lttng-tools 3/3] Fix: do not refer to objects as .o
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The extension of object files is platform dependant,
use $(OBJEXT) instead of .o when referring to objects.

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
Signed-off-by: Jérémie Galarneau <jeremie.galarneau@efficios.com>
---
 tests/regression/tools/live/Makefile.am |  8 +++----
 tests/unit/Makefile.am                  | 38 ++++++++++++++++-----------------
 2 files changed, 23 insertions(+), 23 deletions(-)

diff --git a/tests/regression/tools/live/Makefile.am b/tests/regression/tools/live/Makefile.am
index 24ccc7c..8bc213d 100644
--- a/tests/regression/tools/live/Makefile.am
+++ b/tests/regression/tools/live/Makefile.am
@@ -16,10 +16,10 @@ LIBHASHTABLE=$(top_builddir)/src/common/hashtable/libhashtable.la
 LIBRELAYD=$(top_builddir)/src/common/relayd/librelayd.la
 LIBHEALTH=$(top_builddir)/src/common/health/libhealth.la
 
-LIVE=$(top_builddir)/src/bin/lttng-sessiond/session.o \
-	 $(top_builddir)/src/bin/lttng-sessiond/consumer.o \
-	 $(top_builddir)/src/bin/lttng-sessiond/utils.o \
-	 $(top_builddir)/src/bin/lttng-sessiond/snapshot.o
+LIVE=$(top_builddir)/src/bin/lttng-sessiond/session.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/consumer.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/utils.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/snapshot.$(OBJEXT)
 
 noinst_PROGRAMS = live_test
 EXTRA_DIST = test_kernel test_lttng_kernel
diff --git a/tests/unit/Makefile.am b/tests/unit/Makefile.am
index b698107..3355e59 100644
--- a/tests/unit/Makefile.am
+++ b/tests/unit/Makefile.am
@@ -42,10 +42,10 @@ test_uri_SOURCES = test_uri.c
 test_uri_LDADD = $(LIBTAP) $(LIBCOMMON) $(LIBHASHTABLE)
 
 # Session unit test
-SESSIONS=$(top_builddir)/src/bin/lttng-sessiond/session.o	\
-	 $(top_builddir)/src/bin/lttng-sessiond/consumer.o \
-	 $(top_builddir)/src/bin/lttng-sessiond/utils.o \
-	 $(top_builddir)/src/bin/lttng-sessiond/snapshot.o \
+SESSIONS=$(top_builddir)/src/bin/lttng-sessiond/session.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/consumer.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/utils.$(OBJEXT) \
+	 $(top_builddir)/src/bin/lttng-sessiond/snapshot.$(OBJEXT) \
 	 $(top_builddir)/src/common/libcommon.la \
 	 $(top_builddir)/src/common/health/libhealth.la \
 	 $(top_builddir)/src/common/sessiond-comm/libsessiond-comm.la
@@ -58,18 +58,18 @@ test_session_LDADD += $(SESSIONS)
 
 # UST data structures unit test
 if HAVE_LIBLTTNG_UST_CTL
-UST_DATA_TRACE=$(top_builddir)/src/bin/lttng-sessiond/trace-ust.o \
-	       $(top_builddir)/src/bin/lttng-sessiond/consumer.o \
-	       $(top_builddir)/src/bin/lttng-sessiond/utils.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/buffer-registry.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/ust-registry.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/ust-metadata.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/ust-app.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/ust-consumer.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/fd-limit.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/session.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/snapshot.o \
-		   $(top_builddir)/src/bin/lttng-sessiond/agent.o \
+UST_DATA_TRACE=$(top_builddir)/src/bin/lttng-sessiond/trace-ust.$(OBJEXT) \
+	       $(top_builddir)/src/bin/lttng-sessiond/consumer.$(OBJEXT) \
+	       $(top_builddir)/src/bin/lttng-sessiond/utils.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/buffer-registry.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/ust-registry.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/ust-metadata.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/ust-app.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/ust-consumer.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/fd-limit.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/session.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/snapshot.$(OBJEXT) \
+		   $(top_builddir)/src/bin/lttng-sessiond/agent.$(OBJEXT) \
 		   $(top_builddir)/src/common/libcommon.la \
 		   $(top_builddir)/src/common/health/libhealth.la \
 		   $(top_builddir)/src/common/sessiond-comm/libsessiond-comm.la
@@ -81,9 +81,9 @@ test_ust_data_LDADD += $(UST_DATA_TRACE)
 endif
 
 # Kernel data structures unit test
-KERN_DATA_TRACE=$(top_builddir)/src/bin/lttng-sessiond/trace-kernel.o	\
-		$(top_builddir)/src/bin/lttng-sessiond/consumer.o	\
-		$(top_builddir)/src/bin/lttng-sessiond/utils.o \
+KERN_DATA_TRACE=$(top_builddir)/src/bin/lttng-sessiond/trace-kernel.$(OBJEXT) \
+		$(top_builddir)/src/bin/lttng-sessiond/consumer.$(OBJEXT) \
+		$(top_builddir)/src/bin/lttng-sessiond/utils.$(OBJEXT) \
 		$(top_builddir)/src/common/libcommon.la \
 		$(top_builddir)/src/common/health/libhealth.la \
 		$(top_builddir)/src/common/sessiond-comm/libsessiond-comm.la
-- 
2.7.4

